### ISSUE TITLE
Open/dowload original file from the cloud if changed

### ISSUE THREAD

**Issue by tsroman** [kolyvan/kybook#120]
This is related to #97 we talked. 
How about this, if you try to open/read the file directly from the cloud storage. And the file is changed, for example, different size or modification date. You just download updated file from the storage and replace local automatically, so the user can read the updated version without manually deleting local from DB. But again this is only when you push the button "read" and only from cloud storage. Seems logical for me, because you do not check update every file, but only one that user tries to open. What do you think?

**Comment by kolyvan** [comment]
Sorry but seems you miss the application.
KyBook is actually the reader (or viewer) application but not the editor at all.
It optimized to work with permanent unchanged files.
Typical user has ebooks that never changes and KyBook is perfect for this.

But in your case, most likely you need another application (there are lots of them in App Store) for your goals. KyBook is not the most suitable application for you.

**Comment by tsroman** [comment]
@kolyvan 
Fast closing.
Wtf? Cloud storages completely unchanged, really?

Your answer is bullshit, this is clearly a feature that MUST be implemented.
Actually, before KyBook I used a book reader with this feature. And it works perfectly with local caching.

You already have a feature to remove from database. Why not make a SIMPLE check when user try to read a single file from cloud storage (NOT from local). Because you are not opening local version of the file - check if the file is not changed, this is just so intuitively and logically.

A good way to keep customers, "go find another app." Your app is the most advanced book reader I used till now. Currently, I'm unsatisfied only with this.
